app/src/main/java/com/eventyay/organizer/data/faq/FaqRepositoryImpl.java

```diff
@@ -1,11 +1,12 @@
 package com.eventyay.organizer.data.faq;
 
+import android.content.Context;
```

Comment:
No import of android classes in Repository. Use dependency injection to inject Database

Reply:
Updated.